sopel_modules/chanlogs/__init__.py

```diff
@@ -111,6 +112,17 @@ def _format_template(tpl, bot, trigger, **kwargs):
     return formatted
 
 
+def _channel_is_opted_in(is_channel, channel, opted_in_channels):
```

Comment:
Why does this take `is_channel` as an argument? Callers derive it from `Trigger.sender.is_nick()` anyway, so… that can be part of the function.

Also: `sopel.tools.Identifier`, of which `Trigger.sender` is an instance, doesn't have an `is_channel()` method. Did you…_test_ this patch?